src/main/java/baseball/domain/Game.java

```diff
@@ -0,0 +1,38 @@
+package baseball.domain;
+
+public class Game {
+    int strike;
+    int ball;
+    int[] randomNumbers;
+
+    public void intiField() {
```

Comment:
`init` 으로 고치면 좋을 것 같아요!
저두 오타가 많은데, 인텔리제이 IDE 에서 단어 이상하게 적으면 빨간줄 나오게 되더라구요.
빨간 줄 나올 때 마다 다시 보고 있어요. 🤣

```suggestion
    public void initField() {
```

Reply:
아 오타가 있었군요 ㅠㅠ 확인 감사합니다 !!